app/services/vendor_status.rb

```diff
@@ -49,6 +49,16 @@ def all_phone_vendor_outage?
     all_vendor_outage?(PHONE_VENDORS)
   end
 
+  def gpo_only?
```

Comment:
Should `gpo_only?` and `allow_hybrid_flow?` go in `FeatureManagement`, since they depend on vendor status and configuration? I have some similar code for an `idv_available?` method in #7970 that I put in there for that reason.

Reply:
We renamed this class OutageStatus since the outage could be predicated on a vendor outage or a feature flag config option.

Reply:
Hmm, I think I preferred the separation of `VendorStatus` and `FeatureManagement`. To me it is not necessarily clear what `OutageStatus` is reporting on -- who is having the outage? Is it us or a vendor? 

I think there's a clear separation between:

- `VendorStatus` - which vendors are operational right now? Which _classes_ (e.g. idv) of vendors are having trouble right now
- `FeatureManagement` - given the larger state of the application, including configuration and vendor availability, what features are currently available to users?